README.md

```diff
@@ -259,14 +259,23 @@ In other scenarios, a component may have an initial loading state (e.g. loading
 
 ```javascript
 class SlowElem extends LitElement {
-  render() {
-    return html`<p>I take my time</p>`;
+  constructor() {
+    super();
+    this._loadingCompletePromise = new Promise(
+      resolve => this._loadingCompleteResolve = resolve
+    );
   }
-  async getLoadingComplete() {
-    return new Promise(resolve => {
-      setTimeout(() => resolve(), 2000);
+  connectedCallback() {
+    this._fetchMyData().then(() => {
+      if (this._loadingCompleteResolve) {
+        this._loadingCompleteResolve();
+        this._loadingCompleteResolve = undefined;
+      }
```

Comment:
Is this a pattern we want to encourage, or is it simpler to just resolve regardless?

Reply:
Yeah I'm good either way, it's all a matter of whether we care that it only gets resolved once. But if this is the example, it'll get copied everywhere. :)

Reply:
Oops missed this PR, thanks Dave! Yeah I'm also indifferent, `this._fetchMyData().then(() => this._loadingCompleteResolve())` might have been simpler but 🤷.